### ISSUE TITLE
feature request - color last tabs or number them

### ISSUE THREAD

**Issue by syberkitten** [harshayburadkar/clut-chrome-extension#10]
Love the extension and use it daily,
actually cannot without it :)

Feels like more ques (hints)
as to my last location, by having a color fadeout
for a few minutes or seconds on each tab visited.

this would make life much easier when you have a lot
of open tabs.

should it be hard / easily possible?

**Comment by ORESoftware** [comment]
yes this would be a nice feature, highlight the active tab, and the previous active tab

**Comment by harshayburadkar** [comment]
Sure.
Currently though, I may not have time to look into this. 
If someone has the time I will be happy to discuss and review the PR or approaches.
